svreal/__init__.py

```diff
@@ -1,6 +1,54 @@
+import os
 from pathlib import Path
 
+from svreal.constants import *
+from svreal.enums import RealType
+from svreal.hard_float import real2recfn, recfn2real
+from svreal.fixed_point import real2fixed, fixed2real, calc_fixed_exp
```

Comment:
This file now uses the conversion functions provided by **svreal**, rather than re-implementing them here